egs2/asvspoof5/spk1/local/asvspoof5_train_data_prep.py

```diff
@@ -0,0 +1,48 @@
+# asvspoof5_train_data_prep.py
```

Comment:
FYI, we're releasing the official ASVspoof5 dataset in a few days via zenodo. 
Can you please confirm if the formats match with the official public version?

I believe the version you're using is the one from the challenge.

Reply:
That's nice :)
please share the link when it's ready so I can adapt